ouroboros-consensus/src/ouroboros-consensus/Ouroboros/Consensus/MiniProtocol/ChainSync/Client.hs

```diff
@@ -1988,7 +1978,7 @@ invalidBlockRejector ::
   -> Watcher m
          (WithFingerprint (HeaderHash blk -> Maybe (InvalidBlockReason blk)))
          Fingerprint
-invalidBlockRejector tracer version getIsInvalidBlock getCandidate =
+invalidBlockRejector tracer _version getIsInvalidBlock getCandidate =
```

Comment:
It seems we could remove this parameter.

Reply:
(I agree, modulo my general caveat about the Consensus Layer retaining the ability to run without block diffusion pipelining.)

Reply:
So maybe we want to leave it after all.

Reply:
It still could be removed as testing whether diffusion pipelining is (not) enabled no longer relies on the `NodeToNodeVersion`.

Reply:
fyi, I've left it in case o-network introduced something in a future NodeToNode version and you'd need to distinguish based on it here. The patch now and later is/would be smaller, and you never know when version information could come in handy.